app/home/open-request/controller.js

```diff
@@ -0,0 +1,57 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class OpenRequestController extends Controller {
+  @service store;
+  @service router;
+  @service session;
+
+  @tracked loggedAs;
+  @tracked currentUser;
+  @tracked title;
+  @tracked machine;
+  @tracked type;
+  @tracked record;
+
+  get hasEmptyField() {
+    return !(this.title && this.machine && this.type);
+  }
+
+  @action
+  onTypeChange(event) {
+    this.type = event.target.value;
+  }
+
+  @action
+  onMachineChange(event) {
+    this.machine = event.target.value;
+  }
+
+  @action
+  onTitleChange(event) {
+    this.title = event.target.value;
+  }
+
+  @action
+  async onCreatNewRequest() {
+    this.record = {
+      title: this.title,
+      type: this.type,
+      machine: this.machine,
+      owner: this.session.currentUser,
+    };
+    const createNewRequest = await this.store.createRecord(
+      'request',
+      this.record
+    );
+    await createNewRequest.save();
+    this.redirectionAfterNewRequest();
+  }
+
+  redirectionAfterNewRequest() {
+    this.router.transitionTo('home.requests');
+    return;
+  }
+}
```

Comment:
Overall few things:
1. Consider to move this all logic to provider component, like `<Home::OpenRequest />`
2. After that in `contructor` this new component create empty record like:
```js
constructor() {
  super(...arguments);
  this.record = this.store.createdRecord('request');
}
```
Then you can modify just fields in this `record` model instead created separate @tracked for each them - think about model which can have 30 attributes, then you have to create 30 tracked's 😅

Reply:
Regarding to this.record approach, take an example from ours designer https://github.com/kkrywult1983/CarRental/blob/00aff71b18eae2a8432d608c0f04d1ada084d643/app/settings/controller.js 😄